### ISSUE TITLE
How is the PoseLoss loss function derived?

### ISSUE THREAD

**Issue by Shania-F** [youngguncho/PoseNet-Pytorch#11]
Great implementation of PoseNet! However, the loss doesn't seem to converge to minima when I start learning sx and sq. Any help with recreating the results in the Geometric Pose Loss paper would be a big help!
